tests/run-fvttests.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+cat << EOT >> fvttest.xml
```

Comment:
Eventually all these run-XXXtest.sh files will need to have actual tests in them as this is a requirement for promoting to a CSP environment.

Reply:
Sure. I figured these could be placeholders for now since this will run in CDT only for the initial release.